bin/impl/commands.sh

```diff
@@ -261,19 +261,18 @@ EOF
 }
 
 function uno_status_main() {
-  # TODO this should be converted to using pgrep
   # shellcheck disable=SC2009
-  atmp="$(ps -ef | grep accumulo\\.start | awk '{print $NF "(" $2 ")"}' | tr '\n' ' ')"
+  atmp="$(pgrep -f accumulo\\.start -a | awk '{pid = $1;for(i=1;i<=NF;i++)if($i=="org.apache.accumulo.start.Main")print $(i+1) "("pid")"}' | tr '\n' ' ')"
```

Comment:
Nice!

With the switch to pgrep, we can remove the shellcheck exceptions before the lines that were changed (just remove the line containing the `# shellcheck disable=SC2009`).